packages/tabs/src/Tabs.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+// hixme-ui
+import Text from '@hixme-ui/text'
+import Container from '@hixme-ui/container'
+
+import StyledTab from './StyledTab'
+
+class Tabs extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      tabIndex: 0,
+    }
+  }
+
+  componentDidMount() {
+    // eslint-disable-next-line react/no-did-mount-set-state
+    this.setState({
+      tabIndex: this.props.initWithIndex,
+    })
+  }
+
+  handleTabClick = (clickedIndex) => {
```

Comment:
This syntax is throwing me off as I haven't seen it with a Class, nor can I speak to it.  I checked out the transpiled code and it handles this syntax differently, in that it binds it to `this` so you don't have to explicitly bind `this` to it.  The other Class level functions are transpiled differently.